src/patternfly/components/Table/table.scss

```
@@ -572,6 +573,10 @@
     &.pf-m-selected {
       background-color: var(--#{$table}__tbody--m-selected--BackgroundColor);
       outline-offset: var(--#{$table}__tbody--m-selected--OutlineOffset);
+
+      .#{$table}__tr.pf-m-expanded:not(.pf-v6-c-table__expandable-row) {
+        border: none;
+      }
```

What is this needed for? I don't see any changes when toggling it in dev tools.

```suggestion
      .#{$table}__tr.pf-m-expanded:not(.#{$table}__expandable-row) {
        border: none;
      }
```

---

@mcoker Thank you, that is correct